src/utils/type.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Check type of operand with more specificity than `typeof`.
+ *   Slightly modified version of MDN helper found in `typeof` definition page.
+ * @link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#real-world_usage
+ *
+ * @param {*} obj
+ * @param {*} fullClass
+ * @returns {string}
+ */
+export default function type(obj, fullClass = false) {
```

Comment:
Are we using the `fullClass` boolean? if not, remove it.

Reply:
Removed.